Framework/DataHandling/src/GenerateGroupingPowder.cpp

```diff
@@ -244,11 +244,11 @@ void GenerateGroupingPowder::init() {
   fileExtensionXMLorHDF5->addAllowedValue(std::string("nxs"));
   fileExtensionXMLorHDF5->addAllowedValue(std::string("nx5"));
   declareProperty("FileFormat", std::string("xml"), fileExtensionXMLorHDF5,
-                  "File extension/format for saving output: either xml (default) or nxs/nx5.");
-  declareProperty(std::make_unique<FileProperty>("GroupingFilename", "", FileProperty::Save,
+                  "File extension/format for saving output: either xml (default) or nxs/nx5.", PropertyMode::Optional);
+  declareProperty(std::make_unique<FileProperty>("GroupingFilename", "", FileProperty::OptionalSave,
                                                  std::vector<std::string>{"xml", "nxs", "nx5"}),
                   "A grouping file that will be created.");
-  declareProperty("GenerateParFile", true,
+  declareProperty("GenerateParFile", false,
```

Comment:
Changing the default value generally breaks system tests and user scripts you'll need to change the default value back.